### ISSUE TITLE
Incompatible with NodeJS 10

### ISSUE THREAD

**Issue by vain** [profitbricks/profitbricks-cli#68]
With NodeJS 10, which recently landed in Arch Linux, everything just returns empty lists:

```
$ node --version
v10.0.0

$ profitbricks --version
4.1.0

$ profitbricks datacenter list
Datacenters
$

$ profitbricks --datacenterid $foo server list
Servers
$
```

We saw something similar with NodeJS 8.1, see #60.

Let me know if you can't reproduce this and/or if you need more information to debug.

Thanks!

**Comment by nurfet-becirevic** [comment]
The issue should be fixed in the new release, v4.1.1.

**Comment by vain** [comment]
Looks good, thank you. :)
